blockchain/vm/contracts_test.go

```diff
@@ -21,783 +21,207 @@
 package vm
 
 import (
+	"bytes"
+	"encoding/json"
 	"fmt"
+	"io/ioutil"
 	"math/big"
 	"testing"
 
+	"github.com/klaytn/klaytn/blockchain/state"
 	"github.com/klaytn/klaytn/blockchain/types"
 	"github.com/klaytn/klaytn/blockchain/types/accountkey"
-	"github.com/klaytn/klaytn/crypto"
-	"github.com/stretchr/testify/require"
-
-	"github.com/klaytn/klaytn/blockchain/state"
 	"github.com/klaytn/klaytn/common"
+	"github.com/klaytn/klaytn/crypto"
 	"github.com/klaytn/klaytn/params"
 	"github.com/klaytn/klaytn/storage/database"
+	"github.com/stretchr/testify/require"
 )
 
 // precompiledTest defines the input/output pairs for precompiled contract tests.
 type precompiledTest struct {
-	input, expected string
-	gas             uint64
-	name            string
-	noBenchmark     bool // Benchmark primarily the worst-cases
+	Input, Expected string
+	Gas             uint64
+	Name            string
+	NoBenchmark     bool // Benchmark primarily the worst-cases
```

Comment:
Why did you change these names?

Reply:
I have to capitalize them to import test datas from json file.. Umm, do you have any idea..?

Reply:
It just looked strange because the struct name is not capitalized and the fields are capitalized.
It looks fine to me :)